extensions/amp-sticky-ad/0.1/amp-sticky-ad.css

```diff
@@ -23,6 +23,7 @@ amp-sticky-ad {
   z-index: 2;
   /* TODO(zhouyx, #3250): discuss on what the max-height should be? */
   max-height: 100px !important;
+  margin-bottom: 0 !important;
```

Comment:
The same effect can be done with a transparent background color and `padding-bottom`...